### ISSUE TITLE
Slowness and low fps after using for a few minutes

### ISSUE THREAD

**Issue by caduHD4** [mihonapp/mihon#439]
### Steps to reproduce

After using it for a while the app starts to have low fps, something that didn't happen in other versions 

### Expected behavior

It must be fluid, it had never been like that, not even when it was tachiyomi

### Actual behavior

When I reinstall the latest version it stops causing problems, but after a few minutes...

### Crash logs

App version: 0.16.3 (standard, 139663a, 4, 2024-01-29T20:34Z)
Android version: 11 (SDK 30; build RKQ1.200826.002 test-keys)
Device brand: Xiaomi
Device manufacturer: Xiaomi
Device name: raphael (raphael)
Device model: Mi 9T Pro
WebView: Android System WebView 121.0.6167.178

--------- beginning of system
--------- beginning of main
02-23 01:39:50.997 11505 11505 E app.mihon: Not starting debugger since process cannot load the jdwp agent.
02-23 01:39:51.415 11505 11536 E SQLiteLog: (283) recovered 309 frames from WAL file /data/data/app.mihon/databases/tachiyomi.db-wal

### Mihon version

0.16.3

### Android version

Android 11

### Device

Xiaomi Mi 9t pro / k20 pro

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https://mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[0.16.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by MajorTanya** [comment]
Does this still persist for you on 0.16.4?

**Comment by caduHD4** [comment]
Yes, unfortunately the problem still persists, I don't know what it could be.

**Comment by lhjgege** [comment]
I had the same problem, watching the comics load slowly for a while.Getting out of the application and getting back in is better.

**Comment by caduHD4** [comment]
It happens to me even before I start reading something.

**Comment by lhjgege** [comment]
> @lhjgege mind telling us what phone and OS you are using, maybe wee can find a correlation? 介意告诉我们您正在使用什么手机和操作系统，也许我们可以找到相关性？

I'm using this XiaoMi 13  System is HyperOS 1.0.6

**Comment by plumber-craic** [comment]
I also observe this: as I zoom around a large page or change pages it looks like only a few frames per second. Kinda jarring and takes me out of the story. 
Mihon 0.16.5, Android 13, One UI 5.1.1, Galaxy Tab S7+. Reading a cbz comic via Komga extension.

Clearing the cache seems to help.
